packages/cli/internal/pkg/cli/context/manager.go

```diff
@@ -130,6 +131,11 @@ func (m *Manager) setArtifactUrl() {
 		return
 	}
 	m.artifactUrl, m.err = m.Ssm.GetCommonParameter(artifactParameter)
+	if m.err != nil {
+		if maybeActionableErr := actionableerror.FindSuggestionForError(m.err, actionableerror.AwsErrorMessageToSuggestedActionMap); maybeActionableErr != nil {
+			m.err = maybeActionableErr
+		}
```

Comment:
this can be simplified to the following;
```
m.err = actionableerror.FindSuggestionForError(m.err, actionableerror.AwsErrorMessageToSuggestedActionMap)
```

This is because if actionable error doesn't find a match then it just returns the base error.